app/publishing/delinkedCalculation/schema.js

```diff
@@ -0,0 +1,85 @@
+const Joi = require('joi')
+const { DELINKEDCALCULATION } = require('../types')
+
+const minSbi = 105000000
+const maxSbi = 999999999
+const minFrn = 1000000000
+const maxFrn = 9999999999
+
+module.exports = Joi.object({
+  applicationId: Joi.number().integer().required().messages({
+    'number.base': 'applicationId should be a type of number',
+    'number.integer': 'applicationId should be an integer',
+    'any.required': 'The field applicationId is not present but it is required'
+  }),
+  calculationReference: Joi.number().integer().required().messages({
+    'number.base': 'calculationReference should be a type of number',
+    'number.integer': 'calculationReference should be an integer',
+    'any.required': 'The field calculationReference is not present but it is required'
+  }),
+  sbi: Joi.number().integer().min(minSbi).max(maxSbi).required().messages({
+    'number.base': 'sbi should be a type of number',
+    'number.integer': 'sbi should be an integer',
+    'number.min': `sbi should have a minimum value of ${minSbi}`,
+    'number.max': `sbi should have a maximum value of ${maxSbi}`,
+    'any.required': 'The field sbi is not present but it is required'
+  }),
+  frn: Joi.string().min(minFrn).max(maxFrn).required().messages({
+    'number.min': `frn should have a minimum value of ${minFrn}`,
+    'number.max': `frn should have a maximum value of ${maxFrn}`,
+    'any.required': 'The field frn is not present but it is required'
+  }),
+  paymentBand1: Joi.string().required().messages({
+    'any.required': 'The field paymentBand1 is not present but it is required'
```

Comment:
suggest adding a string.base validation for this through paymentAmountCalculated